src/Bridges/Variable/Variable.jl

```diff
@@ -0,0 +1,18 @@
+module Variable
+
```

Comment:
Please add documentation on the design of variable bridges: what's the intention, how do they work, how do you add a new variable bridge. It's hard to review the code without this.